
    public Throwable getCause() {
        return runtimeException;
    }
}
